src/main/java/mops/infrastructure/adapters/webflow/questionpolladapter/converters/TimespanConverter.java

```diff
@@ -0,0 +1,29 @@
+package mops.infrastructure.adapters.webflow.questionpolladapter.converters;
+
+import lombok.NoArgsConstructor;
+import mops.infrastructure.adapters.webflow.questionpolladapter.dtos.TimespanDto;
+import mops.domain.models.Timespan;
+import org.springframework.core.convert.converter.Converter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+@NoArgsConstructor
+public class TimespanConverter implements Converter<TimespanDto, Timespan> {
+
+    /**
+     * Timespan converter.
+     * @param timespanDto das DTO vom WebFlow
+     * @return Timespan
+     */
+    @Override
+    @SuppressWarnings({"PMD.LawOfDemeter"})
+    public Timespan convert(TimespanDto timespanDto) {
```

Comment:
Hab ich bei meinem Converter auch noch nicht gemacht, aber Error-Handling in irgendeiner Form?